docs/pages/faqs.md

```diff
@@ -4,15 +4,14 @@ This section of the documentation will cover common questions and encounters oft
 
 ## Tooling
 
-:::note "Incorrect yarn version"
+:::note "Package manager issues"
```

Comment:
While this is good to have here, do we wanna generally mentioned this in source installation step, or anywhere else this could cause issues?

Reply:
As in in `CONTRIBUTING.md` ?

Reply:
anywhere where it's relevant, would have to look though docs myself, maybe check how it's done in ssz repo (if at all), might not need it, I never had issues with install or build myself (not a mac users :D)

Reply:
> As in in CONTRIBUTING.md ?

better to be proactive about this, than having users run into the issue and looking for resolutions online, or if they are lucky, they might look at the FAQ first but that depends on the user behavior